network.md

```diff
@@ -30,17 +30,19 @@ The following codes all indicate that the VM ended execution with an exception,
 -   `EVMC_PRECOMPILE_FAILURE` indicates an errors in the precompiled contracts (eg. invalid points handed to elliptic curve functions).
 
 ```k
-    syntax ExceptionalStatusCode ::= "EVMC_FAILURE"
-                                   | "EVMC_INVALID_INSTRUCTION"
-                                   | "EVMC_UNDEFINED_INSTRUCTION"
-                                   | "EVMC_OUT_OF_GAS"
-                                   | "EVMC_BAD_JUMP_DESTINATION"
-                                   | "EVMC_STACK_OVERFLOW"
-                                   | "EVMC_STACK_UNDERFLOW"
-                                   | "EVMC_CALL_DEPTH_EXCEEDED"
-                                   | "EVMC_INVALID_MEMORY_ACCESS"
-                                   | "EVMC_STATIC_MODE_VIOLATION"
-                                   | "EVMC_PRECOMPILE_FAILURE"
+    syntax String ::= StatusCode2String ( StatusCode ) [function, functional, hook(STRING.token2string)]
+
+    syntax ExceptionalStatusCode ::= "EVMC_FAILURE" [token]
+                                   | "EVMC_INVALID_INSTRUCTION" [token]
+                                   | "EVMC_UNDEFINED_INSTRUCTION" [token]
+                                   | "EVMC_OUT_OF_GAS" [token]
+                                   | "EVMC_BAD_JUMP_DESTINATION" [token]
+                                   | "EVMC_STACK_OVERFLOW" [token]
+                                   | "EVMC_STACK_UNDERFLOW" [token]
+                                   | "EVMC_CALL_DEPTH_EXCEEDED" [token]
+                                   | "EVMC_INVALID_MEMORY_ACCESS" [token]
+                                   | "EVMC_STATIC_MODE_VIOLATION" [token]
+                                   | "EVMC_PRECOMPILE_FAILURE" [token]
```

Comment:
Line up the `[token]`. Add a horizontal rule below `syntax String` and `syntax ExceptionalStatusCode`.